vic/vic_run/src/runoff.c

```diff
@@ -48,7 +48,6 @@ runoff(cell_data_struct  *cell,
     size_t                     time_step;
     int                        last_index;
     int                        tmplayer;
-    size_t                     frost_area;
     int                        fidx;
```

Comment:
`fidx` should be a `size_t`, that will take care of these warnings: https://travis-ci.org/UW-Hydro/VIC/jobs/103274507#L171-L183